### ISSUE TITLE
Port for handheld retrogaming consoles

### ISSUE THREAD

**Issue by orsonmmz** [deathkiller/jazz2-native#35]
Hi @deathkiller!

This is not a real issue, but I did not know how else I could reach you.

I just wanted to let you know, that I have managed to port your project to multiple handheld consoles ([PortMaster](https://github.com/christianhaitian/PortMaster/pull/243)) and it delivers tons of fun to many players. I am truly thankful for your effort, it brings back a lot of great memories!

![image](https://github.com/deathkiller/jazz2-native/assets/5319175/629bd6aa-2ae5-4fc4-8d62-e45ab4408db8)

**Comment by deathkiller** [comment]
Hi!
That's great. Did you have to change something or it just worked?

Also, please call it "Jazz² Resurrection". Otherwise, it might give the impression that it's a port of the original game, but it's not, it's a complete reimplementation.

**Comment by orsonmmz** [comment]
I have simply fiddled with the CMake options, so it looks for the game assets in specific directories. I remember I have tried to compile earlier, but it was crashing at the time. Certainly OpenGLES support has been improved since that time :+1: 

I think the community realizes that many of the games (especially PC ones) are not the original games, but reimplementations. I would prefer to call it "Jazz Jackrabbit 2", because it is immediately recognized by many, but if you insist on naming it "Jazz² Resurrection", then I will respect your request.

**Comment by deathkiller** [comment]
Nice. I thought these devices don't have OpenGL ES 3.0 support. I think I've got request for port it on similar devices that didn't have it.

The problem is that it's not 1:1 copy of the original game - some features are missing, some were added and some are implemented differently, so it's inaccurete it call it "Jazz Jackrabbit 2". Official long name of this project is `"Jazz² Resurrection: Open-source reimplementation of Jazz Jackrabbit 2"`. Short name is `"Jazz² Resurrection"`. The same naming is also used for packages of various Linux distros etc., so to be consistent it would be better to call it `"Jazz² Resurrection"` (or use long name) also here. And it's easier for other people to search more information about it.
